8Knot/queries/query_template.py

```diff
@@ -1,26 +1,30 @@
 import logging
-import pandas as pd
-from db_manager.augur_manager import AugurManager
-from app import celery_app
-from cache_manager.cache_manager import CacheManager as cm
-import io
-import datetime as dt
-from sqlalchemy.exc import SQLAlchemyError
+import cache_manager.cache_facade as cf
 
 """
 TODO:
-(1) update QUERY_NAME
-(2) update 'NAME_query' found in function definition and in the function call that sets the 'ack' variable below.
-'NAME' should be the same as QUERY_NAME
-(3) paste SQL query in the query_string
-(4) insert any necessary df column name or format changed under the pandas column and format updates comment
-(5) reset df index if #4 is performed via "df = df.reset_index(drop=True)"
-(6) go to index/index_callbacks.py and import the NAME_query as a unqiue acronym and add it to the QUERIES list
-(7) delete this list when completed
+(1) Rename the query function to something informative. Replace other instances of "NAME_query" with your
```

Comment:
Is there a way to make "NAME_query" a single variable at the top so it only needs to be done in one place?

Reply:
uhhh I think so? I'll check it out. that'd be naming the function via f-strings I think. I'm not sure if that's supported.

Reply:
No, we can't define the name of a function as a variable and then use it to define the function. Having the user ctrl-f to find all instances of the 'query_NAME' thing should be fine though. I'll add that to the template notes.